include/glow/Runtime/RuntimeTypes.h

```diff
@@ -143,6 +143,19 @@ struct DeviceConfig {
       : backendKind(kind), name(name), parameters(parameters) {}
 
   bool hasName() const { return name != ""; }
+
+  void setDeviceMemory(uint64_t memSize) {
+    parameters["deviceMemory"] = std::to_string(memSize);
```

Comment:
do we expect some of the devices do not need to have deviceMemory?

Specifically why is it in the map and not the member of the DeviceConfig.

Reply:
@nickgg could you clarify this : )

Reply:
already moved the deviceMemory out to be a member of the deviceConfig.
Thanks :)